packages/rx_bloc_cli/mason_templates/bricks/feature_login/__brick__/lib/base/models/credentials_model.dart

```diff
@@ -0,0 +1,28 @@
+{{> licence.dart }}
+
+import 'package:equatable/equatable.dart';
+import 'package:json_annotation/json_annotation.dart';
+
+part 'credentials_model.g.dart';
+
+@JsonSerializable()
+class CredentialsModel with EquatableMixin {
+  CredentialsModel({
+    required this.email,
+    required this.password,
+  });
+
+  final String email;
```

Comment:
Please ensure the model properties are documented properly

Reply:
Done.